enterprise/server/remote_execution/containers/podman/podman.go

```diff
@@ -71,6 +73,12 @@ var (
 
 	sociArtifactStoreTarget = flag.String("executor.podman.soci_artifact_store_target", "", "The GRPC url to use to access the SociArtifactStore GRPC service.")
 	sociStoreKeychainPort   = flag.Int("executor.podman.soci_store_keychain_port", 1989, "The port on which the soci-store local keychain service is exposed, for sharing credentials for streaming private container images.")
+	podmanNetwork           = flagtypes.Alias[string]("executor.docker_network", "executor.podman_network")
+	podmanCapAdd            = flagtypes.Alias[string]("docker_cap_add", "podman_cap_add")
+	podmanDevices           = flagtypes.Alias[[]container.DockerDeviceMapping]("executor.docker_devices", "executor.podman_devices")
+	podmanVolumes           = flagtypes.Alias[[]string]("executor.docker_volumes", "executor.podman_volumes")
+	podmanRuntime           = flag.String("executor.podman.runtime", "", "Enables running podman with other runtimes, like gVisor (runsc).")
+	podmanEnableStats       = flag.Bool("executor.podman.enable_stats", false, "Whether to enable cgroup-based podman stats.")
```

Comment:
I feel like it's confusing to have docker flags that affect podman behavior, and podman flags that affect docker behavior... I guess today, the docker flags already affect podman behavior, but I think we should avoid making the situation more confusing, and just continue to reference the `docker` flags for these (might need to use `flagutil.GetValueForFlagName()`)

Reply:
This is a bit tricky. Because the initialization order between podman.go and docker.go, I would need to move these values out of `var` block 🤔 

Let me look into it.

Reply:
I removed the package vars and call flagutil when init the provider.